mathesar_ui/src/systems/table-view/Body.svelte

```diff
@@ -58,22 +60,28 @@
 />
 
 {#key id}
-  <SheetVirtualRows
-    itemCount={$displayableRecords.length}
-    paddingBottom={30}
-    itemSize={getItemSize}
-    itemKey={(index) => recordsData.getIterationKey(index)}
-    let:items
-    let:api
-  >
-    <ScrollAndResetHandler {api} />
-    {#each items as item (item.key)}
-      {#if $displayableRecords[item.index]}
-        <RowComponent
-          style={item.style}
-          bind:row={$displayableRecords[item.index]}
-        />
-      {/if}
+  {#if usesVirtualList}
+    <SheetVirtualRows
+      itemCount={$displayableRecords.length}
+      paddingBottom={30}
+      itemSize={getItemSize}
+      itemKey={(index) => recordsData.getIterationKey(index)}
+      let:items
+      let:api
+    >
+      <ScrollAndResetHandler {api} />
+      {#each items as item (item.key)}
+        {#if $displayableRecords[item.index]}
+          <Row style={item.style} bind:row={$displayableRecords[item.index]} />
+        {/if}
+      {/each}
+    </SheetVirtualRows>
+  {:else}
+    {#each $displayableRecords as displayableRecord}
+      <Row
+        style={{ position: 'relative', height: rowHeightPx }}
```

Comment:
Height of the row should be calculated using `getItemSize` method or rows with different heights would be misaligned eg., Group headers.

Reply:
Thanks. Fixed in c3bbbc14d3d34301cc14499b9d7a43fc0005e947